.github/workflows/tests.yaml

```diff
@@ -20,9 +20,13 @@ jobs:
       matrix:
         include:
           - php-version: "8.1"
-            dependencies: "lowest"
+            dependencies: "lowest" # Pimcore 10.5.0
```

Comment:
As per `composer.json` this would currently be `10.0.0`.

Reply:
Yes, but becuase of other dependencies it does go lower than that. Should we then update composer.json?